### PR TITLE
Updated staking faq

### PR DESCRIPTION
**Pull request by raamb**

### CODE REVIEW

File: packages/stake-dapp/src/Pages/FAQ/content.js [hunk 1]
```diff
@@ -47,10 +47,66 @@ export const metamaskFAQ = [
   },
   {
     question: "I am using Metamask but I am unable to connect on the portal",
-    answer: `<p>Please check the version of your Metamask. The staking DApp works with the stable version of Metamask and not with any beta versions.</p>`,
+    answer: `<p>Please check the version of your Metamask. The staking DApp works with the stable version of Metamask and not with beta versions.</p>`,
```
Comment (Vivek205) [inline packages/stake-dapp/src/Pages/FAQ/content.js:50]: @raamb , do we want to mention the exact version of the Metamask?. Staking dApp has a lot of blockchain operations.

File: packages/stake-dapp/src/Pages/FAQ/content.js [hunk 2]
```diff
@@ -47,10 +47,66 @@ export const metamaskFAQ = [
...
   },
   {
     question: "I connected my Ledger hardware wallet to Metamask but am still unable to connect or stake",
     answer: `<p>If you are using a hardware wallet from ledger connected to metamask, you need to have enabled contract data. Activate contract data in the settings of the app if you want to send ERC-20 tokens.</p>`,
   },
+  {
+    question: "When the Metamask window opens up it does not give me an option to confirm or submit the transaction",
+    answer: `<p><ul>
```
Comment (ksridharbabuus) [inline packages/stake-dapp/src/Pages/FAQ/content.js:58]: Generally we dont place UL elements within the paragraph elements. Is there any warning mentioned in the browser, if not we can go ahead with the current code.

Comment (raamb) [reply packages/stake-dapp/src/Pages/FAQ/content.js]: Done, removed the p tags

File: packages/stake-dapp/src/Pages/FAQ/content.js (hunk 1 above)
Comment (raamb) [reply packages/stake-dapp/src/Pages/FAQ/content.js:50]: @Vivek205 we will stay with stable version as that version number changes as well.

**Pull request merged**